vnext/RNW.props

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation. All rights reserved.
+ Licensed under the MIT License.. 
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <UseBundle Condition=" '$(Configuration)' == 'Debug' ">false</UseBundle>
+    <UseBundle Condition=" '$(Configuration)' == 'Release' ">true</UseBundle>
+    <BundledContent>Bundle\**\*</BundledContent>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(UseBundle)' == 'true' and '$(BundledContent)' != ''">
+    <Content Include="$(BundledContent)">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <DefineConstants Condition=" '$(DefineConstants)' != ''  and '$(UseBundle)' == 'true' ">$(DefineConstants);BUNDLE</DefineConstants>
```

Comment:
Is `DefineConstants` a C# specific property?
If so, place into a separate file, like, `vnext/PropertySheets/React.CSharp.props`

Reply:
No. It's target to both C++ and C#